content/en/docs/multi-center/modules/hosp/labevents.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "labevents"
+linktitle: "labevents"
+weight: 3
+date: 2023-09-18
+description: >
+  Laboratory measurements obtained from patient-derived specimens.
+---
+
+The *labevents* table stores the results of all laboratory measurements made for a single patient.
+These include hematology measurements, blood gases, chemistry panels, and less common tests such as genetic assays. This table also includes COVID-19 diagnostic tests and measurements.
+
+## Links to
+
+* *d_labitems* on `itemid`
+
+## Table columns
+
+Name | Postgres data type
+---- | ----
+`labevent_id` | INTEGER NOT NULL
+`subject_id` | INTEGER NOT NULL
+`hadm_id` | INTEGER
+`specimen_id` | INTEGER NOT NULL
+`itemid` | INTEGER NOT NULL
+`order_provider_id` | VARCHAR(10)
+`charttime` | TIMESTAMP(0)
+`storetime` | TIMESTAMP(0)
+`value` | VARCHAR(200)
+`valuenum` | DOUBLE PRECISION
+`valueuom` | VARCHAR(20)
+`ref_range_lower` | DOUBLE PRECISION
+`ref_range_upper` | DOUBLE PRECISION
+`flag` | VARCHAR(10)
+`priority` | VARCHAR(7)
+`comments` | TEXT
+
+### `labevent_id`
+
+An integer which is unique for every row in the table.
+
+### `subject_id`
+
+{{% include "/static/include/subject_id.md" %}}
+
+### `hadm_id`
+
+{{% include "/static/include/hadm_id.md" %}}
+
+### `specimen_id`
+
+Uniquely denoted the specimen from which the lab measurement was made. Most lab measurements are made on patient derived samples (specimens) such as blood, urine, and so on.
+Often multiple measurements are made on the same sample. The `specimen_id` will group measurements made on the same sample, e.g. blood gas measurements made on the same sample of blood.
+
+### `itemid`
+
+An identifier which uniquely denotes laboratory concepts.
+
+### `order_provider_id`
```

Comment:
is this intentional? there is no *provider* table

Reply:
removed!